client/src/ingredients/attributes.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+
+function RenderValues({ values }) {
+  return (
+    <ul>
+      {values.map(value => {
+        return (
+          <li key={value.value}>
+            <code>{value.value}</code>:{" "}
+            <div dangerouslySetInnerHTML={{ __html: value.description }} />
+          </li>
+        );
+      })}
+    </ul>
+  );
+}
+
+function RenderAttributes({ attributes }) {
+  return (
+    <dl>
+      {attributes.map(attribute => {
+        return (
+          <React.Fragment key={attribute.name}>
+            <dt>
+              <code>{attribute.name}</code>: <i>{attribute.type}</i>
+            </dt>
+            <dd>
+              {/* XXX a div tag in the middle of a dd tag!
```

Comment:
It hurts me to land `XXX` code comments into master. But, I think it's fine for now. We need to let this simmer a bit. 

This is definitely a shortcoming of React and there's no good solution (as pointed out in the github issue). 
It's a fact that all these strings (e.g. `attribute.description`) look like this: `<p>bla bla bla</p>` so we *could* to this:

```
const descriptionSans = attribute.description.slice(3, -3); // or whatever the correct syntax is. 
<p dangerouslySetInnerHTML={{ __html: descriptionSans }} />
```

But I'm not sure what would/should happen if the string is`<p>bla bla bla</p><p>ble ble</p><pre>foo bar</pre>`